packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -123,18 +129,55 @@ const PopoverButton = ({ onClick, disabled, children }: PopoverButtonProps) => {
 };
 
 interface EntryValidationTextProps {
-  status: ReleaseAction['entry']['status'];
   action: ReleaseAction['type'];
+  schema: Schema.ContentType;
+  components: { [key: Schema.Component['uid']]: Schema.Component };
+  entry: ReleaseActionEntry;
 }
 
-const EntryValidationText = ({ status, action }: EntryValidationTextProps) => {
+const EntryValidationText = ({ action, schema, components, entry }: EntryValidationTextProps) => {
   const { formatMessage } = useIntl();
+  const { validate } = unstable_useDocument();
+
+  const getValidationErrors = () => {
+    try {
+      validate(entry, { contentType: schema, components });
+
+      return {};
+    } catch (error) {
+      return getYupInnerErrors(error as ValidationError);
+    }
+  };
+
+  const validationErrors = getValidationErrors();
+
+  if (Object.keys(validationErrors).length > 0) {
+    const validationErrorsMessages = Object.entries(validationErrors)
+      .map(([key, value]) =>
+        formatMessage(
+          { id: `${value.id}.withField`, defaultMessage: value.defaultMessage },
+          { field: key }
+        )
+      )
+      .join(' ');
+
+    return (
+      <Flex gap={2}>
+        <Icon color="danger600" as={CrossCircle} />
+        <Tooltip description={validationErrorsMessages}>
+          <Typography textColor="danger600" variant="omega" fontWeight="semiBold" ellipsis>
+            {validationErrorsMessages}
+          </Typography>
+        </Tooltip>
+      </Flex>
```

Comment:
I think you will need to truncate the error message if it is really long similar to bulk publish:

<img width="667" alt="Screenshot 2024-01-16 at 13 36 39" src="https://github.com/strapi/strapi/assets/26598053/e19e1ba8-3bd3-4718-9add-195c6a4cf357">

Similar to the problem on Simone's PR: https://github.com/strapi/strapi/pull/19188